registry.json

```diff
@@ -969,6 +969,33 @@
       "documentation": "https://polkadot.js.org/api/substrate/rpc.html"
     }
   },
+  {
+    "id": "everscale",
+    "name": "Everscale",
+    "coinId": 396,
+    "symbol": "EVER",
+    "decimals": 9,
+    "blockchain": "Everscale",
+    "derivation": [
+      {
+        "path": "m/44'/396'/0'/0/0"
```

Comment:
Yes, EVER Wallet and Surf use this path (`m/44'/396'/0'/0/<account>`). If it is a problem, we could change it to `m/44'/396'/0'/0'/0'` there, but it may cause some questions from users who use WalletV3 contract, why they have a different address with the same seed.

Reply:
Ok then, it's better to be compatible with other wallets